feed-integration-webapp/src/main/resources/crater.properties

```diff
@@ -0,0 +1,11 @@
+con.content_type = application/json
+con.accept  = application/json
+con.setDoOutput = true
+con.setDoInput = true
+
+crater.currencyid = 13 #for INR
+crater.url = https://payments-dev.lite.mybahmni.in
```

Comment:
same comment about externalizing configuration .. these should be configurable externally ..